### ISSUE TITLE
*Vs Code* launch example not working

### ISSUE THREAD

**Issue by simpert** [TypeStrong/ts-node#688]
In the `README.md` an example of how to use *ts-node* with *Vs Code* will execute code but not debug code which is what the `launch.json` file is for I believe.

Please update the example to use the following configuration which will use *ts-node* to execute and debug the current *typescript* file
### With this config breakpoints will be hit
```
{
        "type": "node",
        "request": "launch",
        "name": "Debug Current Typescript File",
        "args": [
            "${relativeFile}"
        ],
        "runtimeArgs": [ "--nolazy", "-r", "ts-node/register" ],
        "cwd": "${workspaceRoot}",
        "protocol": "inspector",
}
```  
No need to specify `sourcemaps`  

> using `"args": [ "${relativeFile}" ]` will pass the path of current file open in editor.  
> you could specify a file if want with `"args": [ "${workspaceFolder}/build.ts" ]`  

### With example *Code* attachs to transpiled javascript
```
{
    "type": "node",
    "request": "launch",
    "name": "Launch Program",
    "runtimeArgs": [
        "-r",
        "ts-node/register"
    ],
    "args": [
        "${workspaceFolder}/index.ts"
    ]
}
```  

Thank you.

**Comment by igimanaloto** [comment]
Add the $workspaceRoot to your launch config's args: 

```
"args": ["${workspaceRoot}/${relativeFile}"],
```

**Comment by sm1th** [comment]
@igimanaloto `${workspaceRoot}` is deprecated in favor of `${workspaceFolder}`

**Comment by FishOrBear** [comment]
```
        {
            "name": "ts",
            "type": "node",
            "request": "launch",
            "args": [
                "${file}"
            ],
            "runtimeArgs": [
                "-r",
                "ts-node/register"
            ],
            "cwd": "${workspaceFolder}",
            "protocol": "inspector",
            "internalConsoleOptions": "openOnSessionStart"
        }
```

**Comment by cspotcode** [comment]
I have tested the debug configuration in our docs to confirm that it works with breakpoints.
Here is where I did the testing:
https://github.com/TypeStrong/ts-node-repros/tree/vscode-debugging

I tweaked our documentation here: #1466

Closing as I believe no further action is required.  Improvements to our documentation can be submitted as pull requests, and [Discussions](https://github.com/TypeStrong/ts-node/discussions) can be used for more open-ended discussion and tech support around debugging.
